src/errors.d

```diff
@@ -18,6 +18,8 @@ import ddmd.globals;
 import ddmd.root.outbuffer;
 import ddmd.root.rmem;
 
+//debug = printGaggedErrors;
```

Comment:
My points:
1. In the latter, enabling the debug feature needs scrolling and finding the line. In the former, that's much easier to do.
2. Now the debug feature has a good name.

It's equivalent with `LOG` enums those are used in some front-end code. When someone finds them, they can learn about them only once.